nhsn/tests/test_pull.py

```diff
@@ -146,7 +151,7 @@ def test_pull_prelim_nhsn_data_output(self, mock_create_backup, mock_socrata, ca
         # Mock Socrata client and its get method
         mock_client = MagicMock()
         mock_socrata.return_value = mock_client
-        mock_client.get.side_effect = [TEST_DATA, []]
+        mock_client.get.side_effect = [PRELIM_TEST_DATA, []]
```

Comment:
praise: whoops, good find

question: if this wasn't failing, maybe the test should be stricter? Right now, it's comparing column names and if any values are missing. Should be also compare all values for equality in `result` vs `expected`?